### ISSUE TITLE
correlator timeout on fem switching

### ISSUE THREAD

**Issue by mkolopanis** [HERA-Team/hera_corr_cm#26]
Executing `correlator_control.py noise_diode_on` will result in a timeout but all (working) fem switches successfully change to the noise setting.

@jack-h  notes that 
>The noise diode on / off probably has a completely inappropriate timeout, since it has to wait for the monitor to stop, which can take ages

**Comment by jack-h** [comment]
Are you running this routinely? (I don't think this feature is particularly robust (even by my low standards), so if you are super eager to put it in some automated script I'll move it up the priority list)

**Comment by mkolopanis** [comment]
I do not believe anyone is using it routinely at this time. I know there was interested in automating it as part of starting observing but has not been mentioned in a few week.

**Comment by jack-h** [comment]
There are certainly much easier ways to implement quickly for urgent
testing if needed. Or maybe it'll mostly work as advertised
